### PR TITLE
feat: some improvements on glossary BiDi

### PR DESCRIPTION
**Pull request by PassionPenguin**
#### Modification

- the definition of the term. according to what the word bidirectional means, and the only macro referring to this glossary (i.e. bidi-ordering), this should be the property (characteristic) of a text, not the text itself.

#### Addition

- example of a BiDi text
- unicode's support of BiDi text

### CODE REVIEW

File: files/en-us/glossary/bidi/index.md [hunk 1]
```diff
@@ -6,8 +6,16 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-**BiDi** (BiDirectional) refers to a document containing both right-to-left and left-to-right text. Even when both directionalities occur in the same paragraph, the text in each language must appear in its proper directionality.
+**BiDi** (BiDirectional) denotes the characteristic a document has, signifying that both right-to-left (RTL) and left-to-right (LTR) text orientations occurred in one document.
```
Comment (chrisdavidmills) [inline files/en-us/glossary/bidi/index.md:9]:
```suggestion
**BiDi** (BiDirectional) describes a document that contains both right-to-left (RTL) and left-to-right (LTR) text.
```

File: files/en-us/glossary/bidi/index.md [hunk 2]
```diff
@@ -6,8 +6,16 @@ page-type: glossary-definition
...
+
+User Interfaces should design to display BiDi content correctly, but many failed in this. An example is the RTL Hebrew name Sarah: שרה, spelled sin (ש) on the right, resh (ר) in the middle, and heh (ה) on the left, which, many programs would display the Hebrew word in the opposite order.
```
Comment (chrisdavidmills) [inline files/en-us/glossary/bidi/index.md:11]:
```suggestion
User Interfaces should display BiDi content correctly, but many fail. An example is the Hebrew translation of the name Sarah: שרה, spelled (from right-to-left) sin (ש), resh (ר), heh (ה). Many programs would display the letters in the opposite order.
```

File: files/en-us/glossary/bidi/index.md [hunk 3]
```diff
@@ -6,8 +6,16 @@ page-type: glossary-definition
...
+
+It is possible to simply flip the left-to-right display order to a right-to-left display order, but doing this sacrifices the ability to correctly display left-to-right scripts. With bidirectional script support, it is possible to mix characters from different scripts on the same page, regardless of writing direction.
```
Comment (chrisdavidmills) [inline files/en-us/glossary/bidi/index.md:13]:
```suggestion
It is possible to flip the left-to-right display order to a right-to-left display order, but doing this sacrifices the ability to display BiDi scripts correctly — at least one is always incorrectly displayed. With bidirectional script support, it is possible to mix characters from different scripts on the same page, regardless of writing direction.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@PassionPenguin lovely, this is much better, and I thought the example really helped.

I just had a few language suggestions for you.

File: files/en-us/glossary/bidi/index.md [hunk 4]
```diff
@@ -6,8 +6,16 @@ page-type: glossary-definition
...
+
+The Unicode standard provides foundations for complete BiDi support, with detailed rules as to how mixtures of left-to-right and right-to-left scripts are to be encoded and displayed.
 
 ## See also
 
+- {{glossary("Unicode")}}
 - [Bi-directional text](https://en.wikipedia.org/wiki/Bi-directional_text) on Wikipedia
+- [UAX#9: Unicode Bi-directional Algorithm](https://www.unicode.org/reports/tr9/#Resolving_Weak_Types)
```
Comment (chrisdavidmills) [inline files/en-us/glossary/bidi/index.md:21]: Shouldn't this link just point to https://www.unicode.org/reports/tr9/?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@PassionPenguin oop, one more thing I noticed.

File: files/en-us/glossary/bidi/index.md (hunk 4 above)
Comment (PassionPenguin) [reply files/en-us/glossary/bidi/index.md:21]: yeap, possibly just copied the wrong link

**Review by chrisdavidmills — APPROVED** [review]
This looks great! Thanks for your hard work on this @PassionPenguin, and apologies for the delay in merging it.

**Pull request merged**